### PR TITLE
Missing HLS profile view accepts unlimited course IDs

### PR DESCRIPTION
**Pull request by Qubad786**
Missing HLS profile view accepts unlimited course IDs
 -- changed GET to POST and fixed corresponding tests

### CODE REVIEW

File: edxval/views.py [hunk 1]
```diff
@@ -268,10 +268,10 @@ class HLSMissingVideoView(APIView):
...
     authentication_classes = (OAuth2Authentication, SessionAuthentication)
 
-    def get(self, request):
-        courses = request.query_params.getlist('courses')
-        batch_size = int(request.query_params.get('batch_size', 50))
-        offset = int(request.query_params.get('offset', 0))
+    def post(self, request):
+        courses = request.data.get('courses')
+        batch_size = int(request.data.get('batch_size', 50))
+        offset = int(request.data.get('offset', 0))
```
Comment (muhammad-ammar) [inline edxval/views.py]: why we need to convert these to `int`, shouldn't they already be in integer format?

File: edxval/views.py [hunk 2]
```diff
@@ -268,10 +268,10 @@ class HLSMissingVideoView(APIView):
     """
     authentication_classes = (OAuth2Authentication, SessionAuthentication)
 
-    def get(self, request):
-        courses = request.query_params.getlist('courses')
-        batch_size = int(request.query_params.get('batch_size', 50))
-        offset = int(request.query_params.get('offset', 0))
+    def post(self, request):
```
Comment (muhammad-ammar) [inline edxval/views.py]: Please add docstring to explain what we are doing in this POST?

File: setup.py [hunk 3]
```diff
@@ -43,7 +43,7 @@ def load_requirements(*requirements_paths):
     return list(requirements)
 
 
-VERSION = '0.1.25'
+VERSION = '0.1.26'
```
Comment (muhammad-ammar) [inline setup.py]: `GET` to `POST` and `POST` to `PUT` are breaking changes, and as per semantic versioning we should increase the MAJOR version which means `0.1.25` -> `1.1.25`

**Comment by Qubad786** [comment]
@muhammad-ammar updated.

**Pull request merged**